di_website/templates/includes/partials/multi-authors.html

```diff
@@ -2,7 +2,11 @@
 
 <div class="m-authors">
     <h3 class="m-authors__title">
-      {% if first_author and authors|length > 0 %}
+      {% if title and authors|length > 1 %}
+        Participants
```

Comment:
Works well and looks good. Would it make sense to make this line `{{ title }}` instead of `Participants` so it can take the title argument from `audio_visual_media.html`?

Reply:
@akmiller01 great idea, its now updated